src/pages/home/report/ReportActionsView.js

```diff
@@ -397,6 +425,68 @@ class ReportActionsView extends React.Component {
         }
     }
 
+    /**
+     * Scrolls to a specific report action ID
+     */
+    scrollToReportActionID() {
+        this.actionIndexID = _.findIndex(this.sortedReportActions, (
+            ({action}) => parseInt(action.reportActionID, 10) === this.props.reportActionID
+        ));
+
+        if (this.actionIndexID !== -1) {
+            ReportScrollManager.scrollToIndex({index: this.actionIndexID, viewPosition: 0.5});
+            this.setState({selectedReportActionID: this.props.reportActionID});
```

Comment:
**question:** Why set to state what is already available via props?

Reply:
Aa per 1:1 discussion we landed on a better named variable like,
```
<ReportActionsList
    reportActionIDToHighlight={this.state.shouldHighlightReportActionID && this.props.reportActionID}
/>
```
I realized a mistake in the logic so i've updated it to,
```
reportActionIDToHighlight={this.state.shouldHighlightReportActionID && this.props.reportActionID ? this.props.reportActionID : 0}
```